elliottlib/bzutil.py

```diff
@@ -277,7 +288,7 @@ def whiteboard_component(self):
 
         :returns: a string if a value is found, otherwise None
         """
-        marker = r'component:\s*(\S+)'
+        marker = r'^pscomponent:\s*(\S+)'
```

Comment:
I'd keep this unchanged, since there are `component:` labels out there (like https://issues.redhat.com/browse/OCPBUGS-1?jql=labels%20%3D%20component%3Aasync-http-client) and there could be someone / something out there that sets it and we miss it..

Reply:
Makes sense, thanks!